lib/tapioca/compilers/dsl/active_storage.rb

```diff
@@ -0,0 +1,66 @@
+# typed: strict
+# frozen_string_literal: true
+
+begin
+  require "active_record"
+  require "active_storage"
+  require "active_storage/attached"
```

Comment:
I think we need to require `active_storage/reflections` here since `attached` should be handled by the default require:
```suggestion
  require "active_record"
  require "active_storage"
  require "active_storage/reflection"
```
since we want to check that they are extending `ActiveStorage::Reflection::ActiveRecordExtensions::ClassMethods`.